packages/test/src/test-activity-log-interceptor.ts

```diff
@@ -116,7 +121,7 @@ test('Activity Worker logs when activity is cancelled with signal', async (t) =>
   t.not(entry, undefined);
   const { durationMs, ...rest } = entry?.meta ?? {};
   t.true(Number.isInteger(durationMs));
-  t.deepEqual(rest, activityLogAttributes(defaultActivityInfo));
+  t.deepEqual(rest, { ...activityLogAttributes(defaultActivityInfo), logSource: LogSource.worker });
```

Comment:
I wonder if we need to say this is a `logSource`, it's already an attribute on a `log message`.

Alternatives: `component`, `sdkComponent`, `module`. `source` on its own makes me think of a source code / location.

Reply:
Even `subsystem` works for me.

Reply:
I want to avoid single words, especially generic ones (source, module, component) as it is more likely to conflict with user's own meta.

Reply:
`sdkComponent` sounds good to me.